config/presets/cli-params.json

```
@@ -20,6 +20,10 @@
     {
       "@id": "urn:solid-server:default:variable:loggingLevel",
       "@type": "Variable"
+    },
+    {
+      "@id": "urn:solid-server:default:variable:templateFolder",
```

```suggestion
      "@id": "urn:solid-server:default:variable:podTemplateFolder",
```
(there can be different kinds of templates in the future)